roles/pre_tasks/tasks/main.yml

```diff
@@ -49,3 +49,16 @@
     src: inventory.j2
     dest: "{{ tower_setup_dir }}/inventory"
   when: pre_tasks_process_template
+
+# FIXME when installing Tower standalone with the embedded database (i.e. without adding a host to the [database] group) and configuring AH to use the same DB, it fails because the firewall isn't opened for PostgreSQL by the installation playbooks. Not sure yet if it's something we should fix in this role, like:
```

Comment:
This is the part I hesitated to add without prior discussion...

Reply:
Going to do some testing, It worries me if this is not in the default install for AH, but more, should it be included with the regular installer, but will test this in my home lab.

IDEA: include this, but its disabled by default, and if user sets configure_firewall_external_database, then this task fires.

Reply:
So After testing, I was able to install on a fresh rhel machines with no firewall configured and access AH without issue. 1 server is the AH, one server the database.

Reply:
Did you explicitly set the database host in the `[database]` group? This is what I didn't do, and most probably why it failed.

Reply:
yes I set the database in the database group

Reply:
Note, the url will not work I removed private keys and it only works for 30 min at the moment
```
- name: Install Ansible Tower
  hosts: localhost
  become: true
  vars:
    tower_releases_url: https://access.cdn.redhat.com/content/origin/files/
    tower_setup_file: ansible-automation-platform-setup-bundle-1.2.1-1.tar.gz
    tower_database_host: 192.168.122.120
    tower_hosts: []
    tower_ah_hosts:
      - 192.168.122.200
  roles:
    - redhat_cop.tower_utilities.install
    ```